### ISSUE TITLE
8.1.25.5 introduces array index access bug

### ISSUE THREAD

**Issue by PMSeitzer** [eugenemel/maven#198]
In `directinfusionprocessor::assessMatch()`:
```
float fragmentObservedIntensity = f->consensus->intensity_array[y];
```
Needs to check that `y != -1` before this access!

**Comment by PMSeitzer** [comment]
Fixed as of 873348d4f293e5d73a2114bef0977dbf72b12c1b

**Comment by PMSeitzer** [comment]
This single issue was released as a patch release `8.1.25.6`
